pyproject.toml

```diff
@@ -601,6 +601,7 @@ ignore-words-list = """
     cna,
     compiletime,
     coo,
+    Copin,
```

Comment:
This can look like a typo, but it is the name of a contributor: https://github.com/astropy/astropy/blob/cd89a7b4b1893ac3066da9f19e8f379daac16464/docs/credits.rst?plain=1#L523

Reply:
Should we ignore the credits file instead?

Reply:
A follow-up issue then?

Reply:
https://github.com/astropy/astropy/issues/16537